Source/WebKitLegacy/mac/WebView/WebPreferences.mm

```diff
@@ -3058,12 +3058,11 @@ - (void)setMediaRecorderEnabled:(BOOL)flag
 
 - (BOOL)CSSIndividualTransformPropertiesEnabled
 {
-    return [self _boolValueForKey:WebKitCSSIndividualTransformPropertiesEnabledPreferenceKey];
+    return YES;
 }
 
 - (void)setCSSIndividualTransformPropertiesEnabled:(BOOL)flag
 {
```

Comment:
You should have moved this down to the `WebPrivateDeprecated` section, ditto for the WebPreferencesPrivate.h counterpart.

Reply:
Filed https://github.com/WebKit/WebKit/pull/20948.